pyproject.toml

```diff
@@ -100,11 +100,11 @@ graphene-django-optimizer = "~0.8.0"
 #       Python 3.8+ where our direct dependency becomes part of stdlib.
 importlib-metadata = {version = "^4.4", python = "<3.8"}
 # Template rendering engine
-Jinja2 = "~3.0.3"
+Jinja2 = "~3.1.0"
 # Optional data validation of config contexts
 jsonschema = "~4.7.0"
 # Rendering of markdown files to HTML
-Markdown = "~3.3.7"
+Markdown = "~3.4.0"
```

Comment:
Similarly, removes deprecated code (https://github.com/Python-Markdown/markdown/blob/master/docs/change_log/release-3.4.md#previously-deprecated-objects-have-been-removed) but doesn't look like any of those changes should impact Nautobot core.

Reply:
Actually, there are apparently a number of extensions that haven't been updated and still depend on these deprecated features, which caused `mkdocs` to pin to `Markdown < 3.4` at present. So upgrading `Markdown` to 3.4 actually forces `mkdocs` to downgrade from 1.3.1 (which has the pin) to 1.3.0 (which does not).

In short: do we want to prioritize latest `Markdown` or latest `mkdocs`, because we can't have both at this time.

Reply:
https://github.com/mkdocs/mkdocs/issues/2892
https://github.com/Python-Markdown/markdown/issues/1277